### DISCUSSION TITLE
Generating multiphase system

### DISCUSSION THREAD

**Discussion by nitheeshkumar11** [neperfepx/neper#716] [Q&A]
My objective is to generate a two-phase system with different grain morphologies. I generated them and now I need the coordinates and weight of every grain from each phase separately. 
I tried using the -statgroup "coo,w" command but it isn't working.
How can I proceed?

Thanks, in advance.

**Comment by rquey** [comment]
Use `-statcell x,y,z,size,group`, which provides the position, size and phase id of each grain.
